index.html

```diff
@@ -15,6 +16,9 @@
     </script>
 </head>
 <body>
+<p>You will be redirected to the documentation below in 5 seconds. Note
```

Comment:
added noscript tag for clarity. the p tag should only display when javascript is enabled.